src/inmanta/agent/executor.py

```diff
@@ -73,24 +88,30 @@ def connection_lost(self, exc: Exception | None) -> None:
         self._sync_stop()
         self.stopped.set()
 
-    def get_method(self, name: str) -> typing.Callable[[...], typing.Coroutine[typing.Any, typing.Any, object]]:
-        if name == "stop":
-            return self.stop
 
-        async def echo(*args):
-            return list(args)
+class StopCommand(inmanta.protocol.ipc_light.IPCMethod[None, ExecutorContext]):
```

Comment:
Just a suggestion: Would it be better to move all the `IPCMethod`s into a separate module?

Reply:
Probably, this one is a bit special, as it is feature of the server itself.